_posts/2019-06-04-loading-large-image-data-with-dask-array.md

```diff
@@ -1,13 +1,124 @@
+    <tr><th> Type </th><td> uint16 </td><td> numpy.ndarray </td></tr>
+  </tbody></table>
+</td>
+<td>
+<svg width="176" height="181" style="stroke:rgb(0,0,0);stroke-width:1" >
+
+  <!-- Horizontal lines -->
+  <line x1="10" y1="0" x2="80" y2="70" style="stroke-width:2" />
+  <line x1="10" y1="61" x2="80" y2="131" style="stroke-width:2" />
+
+  <!-- Vertical lines -->
+  <line x1="10" y1="0" x2="10" y2="61" style="stroke-width:2" />
+  <line x1="17" y1="7" x2="17" y2="68" />
+  <line x1="24" y1="14" x2="24" y2="75" />
+  <line x1="31" y1="21" x2="31" y2="82" />
+  <line x1="38" y1="28" x2="38" y2="89" />
+  <line x1="45" y1="35" x2="45" y2="96" />
+  <line x1="52" y1="42" x2="52" y2="103" />
+  <line x1="59" y1="49" x2="59" y2="110" />
+  <line x1="66" y1="56" x2="66" y2="117" />
+  <line x1="73" y1="63" x2="73" y2="124" />
+  <line x1="80" y1="70" x2="80" y2="131" style="stroke-width:2" />
+
+  <!-- Colored Rectangle -->
+  <polygon points="10.000000,0.000000 80.588235,70.588235 80.588235,131.722564 10.000000,61.134328" style="fill:#ECB172A0;stroke-width:0"/>
+
+  <!-- Horizontal lines -->
+  <line x1="10" y1="0" x2="55" y2="0" style="stroke-width:2" />
+  <line x1="17" y1="7" x2="62" y2="7" />
+  <line x1="24" y1="14" x2="69" y2="14" />
+  <line x1="31" y1="21" x2="77" y2="21" />
+  <line x1="38" y1="28" x2="84" y2="28" />
+  <line x1="45" y1="35" x2="91" y2="35" />
+  <line x1="52" y1="42" x2="98" y2="42" />
+  <line x1="59" y1="49" x2="105" y2="49" />
+  <line x1="66" y1="56" x2="112" y2="56" />
+  <line x1="73" y1="63" x2="119" y2="63" />
+  <line x1="80" y1="70" x2="126" y2="70" style="stroke-width:2" />
+
+  <!-- Vertical lines -->
+  <line x1="10" y1="0" x2="80" y2="70" style="stroke-width:2" />
+  <line x1="55" y1="0" x2="126" y2="70" style="stroke-width:2" />
+
+  <!-- Colored Rectangle -->
+  <polygon points="10.000000,0.000000 55.850746,0.000000 126.438982,70.588235 80.588235,70.588235" style="fill:#ECB172A0;stroke-width:0"/>
+
+  <!-- Horizontal lines -->
+  <line x1="80" y1="70" x2="126" y2="70" style="stroke-width:2" />
+  <line x1="80" y1="131" x2="126" y2="131" style="stroke-width:2" />
+
+  <!-- Vertical lines -->
+  <line x1="80" y1="70" x2="80" y2="131" style="stroke-width:2" />
+  <line x1="126" y1="70" x2="126" y2="131" style="stroke-width:2" />
+
+  <!-- Colored Rectangle -->
+  <polygon points="80.588235,70.588235 126.438982,70.588235 126.438982,131.722564 80.588235,131.722564" style="fill:#ECB172A0;stroke-width:0"/>
+
+  <!-- Text -->
+  <text x="103.513608" y="151.722564" font-size="1.0rem" font-weight="100" text-anchor="middle" >768</text>
+  <text x="146.438982" y="101.155399" font-size="1.0rem" font-weight="100" text-anchor="middle" transform="rotate(-90,146.438982,101.155399)">1024</text>
+  <text x="35.294118" y="116.428446" font-size="1.0rem" font-weight="100" text-anchor="middle" transform="rotate(45,35.294118,116.428446)">2010</text>
+</svg>
+</td>
+</tr>
+</table>
```

Comment:
For some reason, this looks a little weird to me. Maybe GitHub doesn't know how to render it correctly. Though the result is a little confusing to parse. In particular the numbers to the far right side don't appear to be labeled. It's not entirely clear (at least to me) what they are bringing to the table. That said, this may be a GitHub rendering error. I have not looked closely.

<img width="556" alt="Screen Shot 2019-06-18 at 12 52 44 PM" src="https://user-images.githubusercontent.com/3019665/59703492-f9fae680-91c7-11e9-9b56-70c467bcef21.png">

Reply:
Yeah it may just be GitHub is not doing a good job rendering this. It looks much better using MacDown locally. I don't know if GitHub will still have an issue rendering this on the `gh-pages` branch.

<img width="517" alt="Screen Shot 2019-06-18 at 1 03 39 PM" src="https://user-images.githubusercontent.com/3019665/59704172-7f32cb00-91c9-11e9-9cb4-17aae7d42e3c.png">

Reply:
It'll be fine.  If you have a ruby environment set up locally you can run the following to get the full blog webserver running

```
bundle exec jekyll serve
```